examples/0_basic/7_revoke_vc.rs

```diff
@@ -0,0 +1,192 @@
+  let credential_index: u32 = 5;
+  let status: Status = RevocationBitmapStatus::new(service_url, credential_index).into();
+
+  // Build credential using subject above, status, and issuer.
+  let mut credential: Credential = CredentialBuilder::default()
+    .id(Url::parse("https://example.edu/credentials/3732")?)
+    .issuer(Url::parse(issuer_document.id().as_str())?)
+    .type_("UniversityDegreeCredential")
+    .status(status)
+    .subject(subject)
+    .build()?;
+
+  // Sign the Credential with the issuer's verification method.
+  issuer_document.sign_data(&mut credential, key_pair.private(), "#key-1", ProofOptions::default())?;
+  println!("Credential JSON > {:#}", credential);
+
+  // ===========================================================================
+  // Revocation of the Verifiable Credential.
+  // ===========================================================================
+
+  // Update the RevocationBitmap service in the issuer's DID Document.
+  // This revokes the credential's unique index.
+  issuer_document.revoke_credentials("my-revocation-service", &[credential_index])?;
+
+  // Publish the changes.
+  let alias_output: AliasOutput = client.update_did_output(issuer_document.clone()).await?;
+  let rent_structure: RentStructure = client.get_rent_structure()?;
+  let alias_output: AliasOutput = AliasOutputBuilder::from(&alias_output)
+    .with_minimum_storage_deposit(rent_structure)
+    .finish(client.get_token_supply()?)?;
+  issuer_document = client.publish_did_output(&secret_manager_issuer, alias_output).await?;
+
+  let validation_result = CredentialValidator::validate(
+    &credential,
+    &issuer_document,
+    &CredentialValidationOptions::default(),
+    FailFast::FirstError,
+  );
+
+  // We expect validation to no longer succeed because the credential was revoked.
+  assert!(matches!(
+    validation_result.unwrap_err().validation_errors[0],
+    ValidationError::Revoked
+  ));
+
+  // ===========================================================================
+  // Alternative revocation of the Verifiable Credential.
+  // ===========================================================================
+
+  // By removing the verification method, that signed the credential, from the issuer's DID document,
+  // we effectively revoke the credential, as it will no longer be possible to validate the signature.
+  let original_method: DIDUrl<IotaDID> = issuer_document.resolve_method("#key-1", None).unwrap().id().clone();
+  issuer_document.remove_method(&original_method).unwrap();
+
+  // Publish the changes.
+  let alias_output: AliasOutput = client.update_did_output(issuer_document.clone()).await?;
+  let alias_output: AliasOutput = AliasOutputBuilder::from(&alias_output).finish(client.get_token_supply()?)?;
+  client.publish_did_output(&secret_manager_issuer, alias_output).await?;
+
+  // We expect the verifiable credential to be revoked.
+  let mut resolver: Resolver = Resolver::new();
```

Comment:
nit: Possibly better to use `Resolver<IotaDocument>` here instead. In which case line 176 also needs to be changed to use an `IotaDocument` instead. 

You can decide what you want to do here.